department-of-veterans-affairs/caseflow
# Allow VSOs to place their tasks on hold and end holds early

Resolves #12583

### Description
Fixes error caused by VSOs attempting to place a task on hold.

#### BEFORE
<img width="1095" alt="Screen Shot 2019-11-04 at 6 12 01 PM" src="https://user-images.githubusercontent.com/45575454/68165983-e9f16280-ff2e-11e9-9db6-7bd8cbe9dea9.png">
<img width="1098" alt="Screen Shot 2019-11-04 at 6 13 53 PM" src="https://user-images.githubusercontent.com/45575454/68165987-ee1d8000-ff2e-11e9-8587-76abd876c243.png">


#### AFTER
![ezgif-4-b9aa47dd9e14](https://user-images.githubusercontent.com/45575454/68165750-40aa6c80-ff2e-11e9-88b3-83ab84263739.gif)

### Acceptance Criteria
- [ ]  `verify_task_access` disallows VSOs from creating other task types
- [ ] VSOs can place Informal Hearing Presentation tasks on hold
- [ ] VSOs can end holds for Informal Hearing Presentation tasks

### Testing Plan
1. Log in as our favorite VSO (BILLY_VSO)
2. Navigate to Billy's queue and click on the first Prepare Informal Hearing Presentation task.
3. In the actions dropdown, select "Place on hold"
4. Add a misc number of days and notes and press submit
5. Ensure this completes successfully and the task shows up in the user's on hold tab
6. Click back into the case
7. In the actions dropdown, select "End hold early"
8. Ensure this completes successfully and the task shows up in the user's assigned tab
